docs/getting-started.md

```diff
@@ -107,6 +107,12 @@ Antrea can be deployed in NetworkPolicy only mode to an AWS EKS cluster, and
 enforce NetworkPolicies for the EKS cluster. To deploy Antrea in an EKS cluster,
 please refer to this [guide](/docs/eks-installation.md).
 
+### Deploying Antrea in GKE
```

Comment:
I think let us merge EKS and GKE to one section like: Deploying Antrea in EKS and GKE